source/system/Mutex.ooc

```diff
@@ -56,8 +56,6 @@ MutexGlobal: class extends Mutex {
 	lock: override func { This _globalMutex lock() }
 	unlock: override func { This _globalMutex unlock() }
 	free: static func ~all { This _globalMutex free() }
-
-	GlobalCleanup register(|| This free~all())
```

Comment:
@marcusnaslund  
Code in a class body gets added to its default function, if I am not mistaken.  
That means it is executed with each constructor call.  

I don't know if this is related to the bug you were talking about since this seems like a class that would only be initialized once but it could be, so I just wanted to point it out so there is no confusion :)